### PR TITLE
Fix memory leaks part 2

### PR DESCRIPTION
**Pull request by karliss**
SUMMARY: AddressSanitizer: 255295 byte(s) leaked in 6418 allocation(s).

### CODE REVIEW

File: libr/core/cbin.c [hunk 1]
```diff
@@ -403,6 +403,7 @@ static bool bin_raw_strings(RCore *r, int mode, int va) {
 	}
 	RList *l = r_bin_raw_strings (bf, 0);
 	_print_strings (r, l, mode, va);
+	r_list_free(l);
```
Comment (radare) [inline libr/core/cbin.c]: missing space before (

File: libr/core/cmd_print.c [hunk 2]
```diff
@@ -3031,6 +3031,8 @@ static int cmd_print_blocks(RCore *core, const char *input) {
 			to = t;
 		}
 	}
+	r_list_free(list);
```
Comment (radare) [inline libr/core/cmd_print.c]: missing space

File: libr/core/disasm.c [hunk 3]
```diff
@@ -4615,8 +4618,10 @@ static void ds_print_esil_anal(RDisasmState *ds) {
 					}
 				}
 				ds_comment_end (ds, "");
+				r_list_free(list);
```
Comment (radare) [inline libr/core/disasm.c]: missing space

File: libr/core/disasm.c [hunk 4]
```diff
@@ -4615,8 +4618,10 @@ static void ds_print_esil_anal(RDisasmState *ds) {
...
 				break;
 			} else {
+				r_list_free(list);
```
Comment (radare) [inline libr/core/disasm.c]: same

File: libr/core/cbin.c (hunk 1 above)
Comment (karliss) [reply libr/core/cbin.c]: Wasn't `git diff master..mybranch | ./sys/clang-format-diff.py -p1` supposed to catch that? It detected at least one other case.

**Comment by radare** [comment]
dunno i always indent the code by hand. all the function calls have a space before the (, the function signature definitions does not.

File: libr/core/cbin.c (hunk 1 above)
Comment (karliss) [reply libr/core/cbin.c]: Done.

File: libr/core/cmd_print.c (hunk 2 above)
Comment (karliss) [reply libr/core/cmd_print.c]: Done.

File: libr/core/disasm.c (hunk 4 above)
Comment (karliss) [reply libr/core/disasm.c]: Done.

File: libr/core/disasm.c (hunk 3 above)
Comment (karliss) [reply libr/core/disasm.c]: Done.

**Pull request merged**